deepset_cloud_sdk/cli.py

```diff
@@ -128,15 +129,25 @@ def login() -> None:
     Example:
     `deepset-cloud login`
 
-    This prompts you to provide your deepset Cloud API key and workspace name.
+    This prompts you to provide your deepset Cloud API key, workspace name, and environment (eu/us).
```

Comment:
```suggestion
    This prompts you to provide your deepset Cloud API key, workspace name, and environment.
```

You get the list of environments later in the prompt.